CHANGELOG

```diff
@@ -1,5 +1,8 @@
 # Changelog for Hedis
 
+## 0.16.0
+* PR #31. Introduced Haskell Flakesi support in Hedis.
+
```

Comment:
A CHANGELOG might not be necessary here because the libraries that consume hedis will not see any difference in the functionality.

Reply:
Should I remove this then ?

Reply:
Yes